docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,50 @@
+# AntennaAttributionDocumentReporter
+
+The AntennaAttributionDocumentReporter generates an attribution document in PDF format by leveraging the [Eclipse 
+Antenna][1] project. The attribution document can be customized by providing a template bundle JAR, or a directory with 
+template PDF files. 
+
+There is a guide in the [Antenna project documentation][2] which describes how such a template bundle JAR file for the 
+customization has to be created before using the reporter. 
+
+In addition to using a template bundle JAR, there is an alternative way to customize the attribution document. For this, 
```

Comment:
Would it make sense to state whether these two options are similarilz powerful and or for which use cases each of the options is preferable?

Reply:
Actually, both methods are equally powerful but the template bundle JAR option will probably preferred from developers and the directory option from non-developers.